dev-haskell/ghcid/ghcid-0.8.7.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Commit message is missing colon: `dev-haskell/ghcid: bump to 0.8.7`. Otherwise fine to merge.

Reply:
No problem i can fix this.